streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -372,48 +373,112 @@ public void testAddStateStoreWithSink() {
 
     @Test
     public void shouldNotAllowToAddStoresWithSameName() {
+        final StoreBuilder<KeyValueStore<Object, Object>> otherBuilder =
```

Comment:
Just refactoring:
 - add missing generic types
 - use `assertThrows`
 - add check for error message